frontend/stepwell-explorer#23/index.html

```
@@ -0,0 +1,146 @@
+
+  <!-- Detail Modal -->
+  <div class="modal-overlay" id="stepwell-modal" role="dialog" aria-modal="true" aria-labelledby="modal-title" aria-hidden="true">
+    <div class="modal-content">
+      <button class="modal-close" id="modal-close-btn" aria-label="Close stepwell details">
+        <i class="fas fa-times" aria-hidden="true"></i>
+      </button>
+      <div class="modal-header">
+        <h2 id="modal-title" class="modal-title"></h2>
+        <span id="modal-location" class="modal-location"></span>
+      </div>
+      <img id="modal-image" class="modal-image" src="" alt="">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)